Dockerfile.alpine

```diff
@@ -60,7 +53,9 @@ RUN apk update \
     automake \
     cmake \
     g++ \
+    gcompat \
     git \
+    linux-headers \
```

Comment:
@wsobel, do you need `linux-headers` for some reason? When I tested the Alpine image (before this PR was made), I didn’t need to in order to make it work. :thinking:

Reply:
Linux-headers are required for boost. It will not build without it.